commit d4e3d5d241c1c4c99aa61d49d4cb07e646bf815f (refs/changes/58/5358/2)
Author: John Levon <john.levon@joyent.com>
Date:   2019-01-14T15:48:18+00:00 (9 months ago)
    
    OS-7512 git pbchk needs updates after python3 changes
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/usr/src/tools/onbld/Checks/DbLookups.py b/usr/src/tools/onbld/Checks/DbLookups.py
index 1173c12eae..955206750f 100644
--- a/usr/src/tools/onbld/Checks/DbLookups.py
+++ b/usr/src/tools/onbld/Checks/DbLookups.py
@@ -26,12 +26,12 @@
 
 # Copyright 2010, Richard Lowe
 # Copyright 2018 OmniOS Community Edition (OmniOSce) Association.
+# Copyright (c) 2019, Joyent, Inc.
 
 #
 # Various database lookup classes/methods, i.e.:
-#     * monaco
-#     * bugs.opensolaris.org (b.o.o.)
 #     * redmine (illumos.org)
+#     * smartos
 #
 
 import re
@@ -82,11 +82,11 @@ class BugDB(object):
 
 	def __smartosbug(self, cr):
 		url = "http://smartos.org/bugview/json/%s" % cr
-		req = urllib2.Request(url)
+		req = Request(url)
 
 		try:
-			data = urllib2.urlopen(req)
-		except urllib2.HTTPError, e:
+			data = urlopen(req)
+		except HTTPError, e:
 			if e.code == 404 or e.code == 403 or e.code == 400:
 				raise NonExistentBug(cr)
 			else:
